### PR TITLE
Sensivitiy calculation can acccept multiple outputs

### LINKED ISSUE #161: `sensitivityTimeProfiles`: improve text size
It looks weird as many things do not fit in the figure

image

- The title
- Parameter paths (maybe multi-line?)
- Axis tics labels (e.g. the last 1500)

### LINKED ISSUE #162: `sensitivityTimeProfiles`: better color selection
With the current color gradient, it is not clear in which "direction" the colors move as the edges of the gradient look like they are "connected".

image

### LINKED ISSUE #175: `SensitivityAnalysis`: add tests for multiple `OutputPaths`
Currently, we have no tests for this. 

@PavelBal If you can prepare an example with multiple output paths, I can add these tests. 

For the next PR.

### LINKED ISSUE #206: sensitivitySpiderPlot fails for multiple outputs
```
simPath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
simulation <- loadSimulation(simPath)
outputPaths <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
parameterPaths <- c(
  "Aciclovir|Lipophilicity",
  "Applications|IV 250mg 10min|Application_1|ProtocolSchemaItem|Dose",
  "Neighborhoods|Kidney_pls_Kidney_ur|Aciclovir|Glomerular Filtration-GFR|GFR fraction"
)

outputPaths <- c("Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)",
                 "Organism|Age",
                 "Organism|ArterialBlood|Plasma|Aciclovir")
ls_results <- sensitivityCalculation(
  simulation = simulation,
  outputPaths = outputPaths,
  parameterPaths = parameterPaths,
  pkDataFilePath = "../Sensitivities.xlsx"
)

> sensitivitySpiderPlot(sensitivityCalculation = ls_results)
Error in seq.default(0, max(data$PercentChangePK) + 100, 100) : 
  'to' must be a finite number
In addition: Warning message:
In Summary.unit(list(list(0, NULL, 8L), list(1, list(list(1, list( :
 
 Error in seq.default(0, max(data$PercentChangePK) + 100, 100) : 
'to' must be a finite number
10.
stop("'to' must be a finite number")
9.
seq.default(0, max(data$PercentChangePK) + 100, 100)
8.
seq(0, max(data$PercentChangePK) + 100, 100)
7.
check_breaks_labels(breaks, labels)
6.
continuous_scale(ggplot_global$y_aes, "position_c", identity, 
name = name, breaks = breaks, n.breaks = n.breaks, minor_breaks = minor_breaks, 
labels = labels, limits = limits, expand = expand, oob = oob, 
na.value = na.value, trans = trans, guide = guide, position = position, ...
5.
scale_y_continuous(breaks = seq(0, max(data$PercentChangePK) + 
100, 100), labels = as.character(seq(0, max(data$PercentChangePK) + 
100, 100))) at sensitivitySpiderPlot.R#119
4.
.createSpiderPlot(.x, xAxisLog = xAxisLog, yAxisLog = yAxisLog)
3.
.f(.x[[i]], ...)
2.
purrr::map(.x = data %>% split(.$OutputPath), .f = ~.createSpiderPlot(.x, 
xAxisLog = xAxisLog, yAxisLog = yAxisLog)) at sensitivitySpiderPlot.R#70
1.
sensitivitySpiderPlot(sensitivityCalculation = ls_results)

```

### LINKED ISSUE #208: Improve performance for `sensitivitySpiderPlot()`
Currently, `sensitivitySpiderPlot()` has Y-axis labels in the increments of 100, and although this works fine with the example dataset, this can become quite the performance hurdle if these values are anywhere in the thousands since printing all these labels will be an expensive operation, and it will additionally clutter the plot. 

Screenshot 2022-04-09 at 16 02 38

Just go with the default labelling provided by _ggplot2_ and only have a label for the line at y = 100.

### LINKED ISSUE #209: Use new validation helpers from `{ospsuite.utils}`
I had introduced some new validation helpers, especially for sensitivity analysis, but they are currently not being used.

### PR DESCRIPTION
**Pull request by PavelBal**
- [x] closes #206 
- [x] closes #175 
- [x] closes #161 
- [x] closes #162 
- [x] closes #208
- [x] closes #209
- [x] tests for plotting when multiple output paths are specified
- [x] interlink all sensitivity analysis function by using family
- [x] add returns tags to docs
- [x] update website pages for the doc changes

### CODE REVIEW

File: R/sensitivityCalculation.R [hunk 1]
```diff
@@ -61,6 +61,10 @@ sensitivityCalculation <- function(simulation,
   variationRange <- .validateVariationRange(variationRange)
 
   # creating `SimulationResults` batch ------------------------
+  # Store old simulation outputs and set user defined
+  oldOutputSelections <- simulation$outputSelections$allOutputs
```
Comment (PavelBal) [inline R/sensitivityCalculation.R:65]: This was present in the original implementation but has been removed for some reason.

Comment (IndrajeetPatil) [reply R/sensitivityCalculation.R:65]: Like you had told me to do, I completely discarded the original implementation and started afresh on my own.

The PR should have been merged after #175 was resolved. It'd have caught that this part was missing.

**Review by IndrajeetPatil — APPROVED** [review]

**Comment by IndrajeetPatil** [comment]
This PR is designed to solve multiple issues, just the same way [the PR](https://github.com/esqLABS/esqlabsR/pull/156) that originally implemented this functionality.

File: R/zzz.R [hunk 2]
```diff
@@ -1,4 +1,4 @@
 .onLoad <- function(...) {
-  options(warnPartialMatchDollar = TRUE)
+  options(warnPartialMatchDollar = TRUE, warn = -99)
```
Comment (PavelBal) [inline R/zzz.R:2]: What does that mean?

Comment (IndrajeetPatil) [reply R/zzz.R]: This is outdated. That's what I meant when I said suppressing warnings globally.

We don't use that trick anymore.

**Pull request merged**